src/jaegertracing/Config.cpp

```diff
@@ -92,4 +92,24 @@ Config::parsePropagationFormat(std::string strPropagationFormat)
     return propagationFormat;
 }
 
+std::vector<Tag> Config::parseTags(const YAML::Node& node) {
+    if (!node.IsDefined() || !node.IsMap()) {
+        return std::vector<Tag>();
+    }
+    std::vector<Tag> tags;
+    for (auto it=node.begin(); it != node.end(); it++) {
+        if (it->second.IsScalar()) {
+          std::string key = it->first.as<std::string>();
+          std::string value = it->second.as<std::string>();
+          tags.emplace_back(key, value);
+        }
+    }
+    return tags;
+}
+
+void Config::setTag(const std::string& key, Tag::ValueType&& value)
```

Comment:
I think the build is still failing (on Ubuntu).

Is this method even needed in this PR? If the goal is to allow tags configured via YAML, then it's already taken care of in parseTags.

If the method is needed as a programmatic way of setting tags, does the value need to be `Tag::ValueType&&`? The Span uses `opentracing::Value&` https://github.com/jaegertracing/jaeger-client-cpp/blob/4d264a42f54593fdea17ab3063e373e957d25cb3/src/jaegertracing/Span.h#L167-L168
Or we could just restrict to accepting string values only.

Reply:
I think its a "nice to have", but for the main purpose of this PR that's totally fine to remove this for now. Will do the modification soon.